cloud-aws-gov/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsGovConnector.java

```diff
@@ -0,0 +1,15 @@
+package com.sequenceiq.cloudbreak.cloud.aws;
+
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.cloud.aws.common.AwsConstants;
+import com.sequenceiq.cloudbreak.cloud.model.Variant;
+
+@Service
+public class AwsGovConnector extends AwsNativeConnector {
+
+    @Override
+    public Variant variant() {
+        return AwsConstants.AwsVariant.AWS_GOV_VARIANT.variant();
```

Comment:
is the AWS_GOV variant also NATIVE?
from `MultiAzValidator` class below I assume yes

Reply:
yes